index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <div class="header">
+      <a class="mylogo">My Logo</a>
+      <div class='nav'>
+        <img class='line' src="data:," alt>
+        <img class='line' src="data:," alt>
+	<img class='line' src="data:," alt>
```

Comment:
- [ ] Please try to make sure the hamburger button looks similar to that of the Figma template as requested by the previous reviewer. Currently, the lines are too thick. It would be better if you use an image instead of making each line 👌